google/cloud/forseti/scanner/audit/retention_rules_engine.py

```diff
@@ -28,7 +28,7 @@
 
 LOGGER = logger.get_logger(__name__)
 
-SUPPORTED_RETENTION_RES_TYPES = frozenset(['bucket'])
+SUPPORTED_RETENTION_RES_TYPES = frozenset(['bucket', 'bigquery_table'])
```

Comment:
nit: can you use the resource constants to reduce duplication?

https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/google/cloud/forseti/scanner/audit/location_rules_engine.py#L33